src/styl/_settings/_color.styl

```diff
@@ -56,7 +66,7 @@ $color-background-4-dark ?= darken($color-background-4, 79)
 // Weight: 2
 //
 // Styleguide: Design.Colors.Alerts
-$color-info ?= $color-brand
```

Comment:
So.. `$color-brand` is definitely removed ?

Reply:
I wasn't used, `--color-brand` was never setted…
Maybe we should find a way to do it later

Reply:
Well.. yeah it was used.. Here and to set `$color-theme-default`..

Reply:
So now, you dissociate the `color-info` from the `brand-color` right ?
Sure we want to do that ?